website/docs/reference/artifacts/other-artifacts.md

```diff
@@ -22,3 +22,21 @@ This file is used to store a compressed representation of files dbt has parsed.
 **Produced by:** commands supporting [node selection](/reference/node-selection/syntax)
 
 Stores the networkx representation of the dbt resource DAG.
+
+### graph_summary.json
+
+<VersionBlock lastVersion="1.5">
+
+:::info New functionality
+This functionality is new in v1.6. You should upgrade to 1.6 or later if you want to use this artifact.
```

Comment:
for consistency, sugg following what we've done previously https://github.com/search?q=repo%3Adbt-labs%2Fdocs.getdbt.com%20new%20functionality&type=code

```suggestion
This functionality is new in v1.6. 
```